window-management/resources/helpers.js

```diff
@@ -34,7 +34,7 @@ async function buttonClick(test, innerHTML) {
 
 // Grants `window-management` permission and caches `window.screenDetails`.
 async function setUpWindowManagement(test) {
-  assert_true(
+  assert_implements(
```

Comment:
It's a bit sad how `assert_implements()` is really just a fancy `assert_true` but this looks consistent with other usage.